_posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc

```diff
@@ -966,7 +996,7 @@ variable "name_prefix" {
 }
 
 variable "aws_region" {
-  description = "The AWS Region to use as the global config recorder and seed region for AWS GuardDuty."
+  description = "The AWS Region to use as the global config recorder and seed region for GuardDuty."
```

Comment:
AWS does not approve 😡 🤣

Reply:
`:%s/AWS //g`

Reply:
I think I'm missing the joke?

Reply:
I guess officially it's called Amazon GuardDuty not AWS GuardDuty... Though the change above was because I copy/pasted updated variables from `module-security` and some of them had different descriptions...

Reply:
You can ignore this. I was only referencing how much of a stickler AWS is when it comes to their naming/branding. Dropping the "Amazon" or "AWS" prefix will get your talk banned from re:Invent.

Reply:
Ah, then maybe my silly joke should be...
`:%s/[AWS|Amazon] //g`

Reply:
Ah, roger 😁